src/coins.ts

```diff
@@ -1420,6 +1420,20 @@ export const findWrappedGasOnChain = (chainKey: ChainKey): Token => {
   return token
 }
 
+export const findTokenByAddress = (tokenAddress: string): Token | null => {
```

Comment:
`findTokenByAddress` should also contain a `chainId` because the token addresses are always bound to a chain. I have not yet experienced that two token on two different chains had the same Id but we should prepare for that to be save.

Reply:
Ah yes Adrian actually mentioned that already 😅